torch_ttnn/passes/constant_folding_pass.py

```diff
@@ -27,7 +29,7 @@ def call(self, gm: torch.fx.GraphModule):
 
     def _can_fold(self, gm: torch.fx.GraphModule, node):
         for arg in node.args:
-            if isinstance(arg, torch.fx.Node):
+            if not isinstance(arg, (int,)) and isinstance(arg, torch.fx.Node):
```

Comment:
Maybe also float?

Reply:
yes, added in e689c4ec22690079a08630854c7b5de93f3e2fa3.